FutureNorthants/northants-website
# Add media_revisions_ui extension

Add the media_revisions_ui extension so that revisions tab can be seen from the Media > edit screen. 

Also updated allowed_formats to latest version and removed the patch. 

## To Test
- Checkout this branch and then run `./drush.wrapper @docker cache:rebuild` then `./drush.wrapper @docker cim`
- Edit an image or file in Content > Media ensuring the 'Create new revision' checkbox is checked. Enter some text in the 'Revision log message' input then save. 
- Edit the image again and click on the 'Revisions' tab and note that the log of revisions is present, along with the log message.
